metabase/metabase
# Show verified and official statuses in data picker search

Search results in the data selector don't show if items are [verified](https://www.metabase.com/docs/latest/users-guide/06-sharing-answers.html#question-moderation) or [official](https://www.metabase.com/docs/latest/users-guide/collections.html#official-collections). Data selector uses a custom search result component that doesn't support everything the main search does. This PR replaces it with a common search result component supporting everything we need 👍 

### To Verify

1. Spin up Metabase Enterprise
2. Follow [the docs](https://www.metabase.com/docs/latest/users-guide/06-sharing-answers.html#question-moderation) to verify a question
3. Follow [the docs](https://www.metabase.com/docs/latest/users-guide/collections.html#official-collections) to create an official collection and put a few questions there
4. Ask a question > Simple / Custom question
5. Make a few searches, the results should look the same as if you use the navbar search

### Demo

**Before**

<img width="309" alt="before" src="https://user-images.githubusercontent.com/17258145/140550896-0abf9d26-6cad-4ac0-bd15-a4488b947401.png">

**After**

<img width="316" alt="after" src="https://user-images.githubusercontent.com/17258145/140550916-7f1ebb11-25c6-4f4a-bf9e-e2aa4f87e442.png">
